docs/assembly.rst

```diff
@@ -699,7 +694,7 @@ The following assembly will be generated::
         mstore(ret, r)
         return(ret, 0x20)
       }
-      default: { jump(invalidJumpLabel) }
+      default: { revert(0, 0) }
```

Comment:
Then we need an `assert` in the initial solidity code example, and I guess that would make it more complicated.